### PR TITLE
Add standard files

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Add in standard files like SUPPORT.md, DEVELOPMENT.md, etc

### Motivation

Better standardization of repositories

### Review checklist (to be filled by reviewers)

- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] Feature or bugfix must have tests
- [ ] Git history [must be clean](https://github.com/DataDog/apigentools/blob/master/CONTRIBUTING.md#commit-messages)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# CHANGELOG
```
Comment (nmuesch) [inline CHANGELOG.md]: We can retroactively add this manually, or start doing the auto changelog updates for the next release

Comment (nmuesch) [reply CHANGELOG.md]: Actually, I forgot this was in docs/changelog.md, so I moved that here.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Rules are matched bottom-to-top, so one team can own subdirectories
+# and another team can own the rest of the directory.
+
+# All your base
+*                       @DataDog/integrations-tools-and-libraries
+
+# Documentation
+*.md                    @DataDog/baklava @DataDog/integrations-tools-and-libraries
+LICENSE                 @DataDog/baklava @DataDog/integrations-tools-and-libraries
+LICENSE-3rdparty.csv    @DataDog/baklava @DataDog/integrations-tools-and-libraries
```
Comment (gzussa) [inline .github/CODEOWNERS]: You can remove this line

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,41 +1,58 @@
-# Contributing
+# How to contribute
```
Comment (gzussa) [inline CONTRIBUTING.md]: Let's keep `Contributing` for consistency with other repos.

File: RELEASING.md [hunk 4]
```diff
@@ -1,22 +1,40 @@
-# Releasing apigentools
+# Releasing
 
-This document summarizes the process of doing a new release of apigentools.
+This document summarizes the process of doing a new release of this project.
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+This project does not have a strict release schedule. However, we would make a release at least every 6 weeks.
```
Comment (gzussa) [inline RELEASING.md]: let's change this to 2 months

File: apigentools/cli.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Unless explicitly stated otherwise all files in this repository are licensed
 # under the 3-clause BSD style license (see LICENSE).
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
-# Copyright 2019 Datadog, Inc.
+# Copyright 2020 Datadog, Inc.
 import argparse
```
Comment (gzussa) [inline apigentools/cli.py]: I am adding line breaks after the header for all files.

File: .github/workflows/workflows/labeler.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+on:
+- pull_request
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (gzussa) [inline .github/workflows/workflows/labeler.yml]: line break

File: CONTRIBUTING.md (hunk 3 above)
Comment (nmuesch) [reply CONTRIBUTING.md]: Aren't we standardizing to `How to contribute` 
https://github.com/DataDog/datadogpy/pull/503/files#diff-6a3371457528722a734f3c51d9238c13R1

File: apigentools/cli.py (hunk 5 above)
Comment (nmuesch) [reply apigentools/cli.py]: I personally prefer no line break, but don't have a strong opinion if we want them

File: .github/workflows/workflows/labeler.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
```
[the remark this answers is not in the document]
Comment (nmuesch) [reply .github/workflows/workflows/labeler.yml]: 🤦‍♂ Nice catch. Bad copy paste

File: CONTRIBUTING.md (hunk 3 above)
Comment (gzussa) [reply CONTRIBUTING.md]: nevermind 🤦‍♂

File: LICENSE [hunk 8]
```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of integrations-core nor the names of its
+* Neither the name of apigentools nor the names of its
```
Comment (gzussa) [inline LICENSE]: You can also set `the copyright holder`

File: LICENSE [hunk 9]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2019, Datadog, Inc.
+Copyright (c) 2020, Datadog, Inc.
```
Comment (gzussa) [inline LICENSE]: I believe this should also be `2019-Present`

File: LICENSE (hunk 8 above)
Comment (nmuesch) [reply LICENSE]: Not sure I understand, what should I change?

Comment (gzussa) [reply LICENSE]: `apigentools ` => `the copyright holder`

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,32 +1,47 @@
...
+  * `changelog/Changed` for changes in existing functionality.
+  * `changelog/Deprecated` for soon-to-be removed features.
+  * `changelog/Removed` for now removed features.
+  * `changelog/Fixed` for any bug fixes.
+  * `changelog/Security` in case of vulnerabilities.
+  * `changelog/no-changelog` in case this PR should not appear in the changelog at all.
+
+See [here][4] for more details about changelogs.
 
 Your pull request must pass all CI tests before we will merge it. If you're seeing
```
Comment (zippolyte) [inline CONTRIBUTING.md]:
```suggestion
Your pull request must pass all CI tests. If you're seeing
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -46,11 +63,49 @@ The commit message should describe the reason for the change and give extra deta
...
+| `documentation`               | Issues and Pull Requests | Documentation related changes.
+| `duplicate`                   | Issue Only               | Duplicate issue.
+| `invalid`                     | Issue Only               | Invalid issue.
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
```
Comment (zippolyte) [inline CONTRIBUTING.md]:
```suggestion
| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
| `stale/exempt`                | Issues and Pull Requests | Exempt from being marked as stale.
```

File: RELEASING.md [hunk 12]
```diff
@@ -1,22 +1,40 @@
...
-This document summarizes the process of doing a new release of apigentools.
+This document summarizes the process of doing a new release of this project.
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+This project does not have a strict release schedule. However, we would make a release at least every 2 months.
+  - No release will be done if no changes got merged to the `master` branch during the above mentioned window.
+  - Releases may be done more frequently than the above mentioned window.
+  - Create a pager duty schedule corresponding to this.
+  - Create a google calendar schedule corresponding to this.
```
Comment (zippolyte) [inline RELEASING.md]: remove

File: apigentools/__init__.py [hunk 13]
```diff
@@ -1,7 +1,7 @@
 # Unless explicitly stated otherwise all files in this repository are licensed
 # under the 3-clause BSD style license (see LICENSE).
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
-# Copyright 2019 Datadog, Inc.
+# Copyright 2019-Present, Inc.
```
Comment (zippolyte) [inline apigentools/__init__.py]:
```suggestion
# Copyright 2019-Present Datadog, Inc.
```

Comment (zippolyte) [reply apigentools/__init__.py]: everywhere too

Comment (nmuesch) [reply apigentools/__init__.py]: 🤦‍♂

**Pull request merged**